site/src/components/markdown.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Markdown
+description: An element to render html generated from markdown.
+template: standard
+---
+
+# Markdown
+
+The markdown element provides bare element styling to lists, paragraphs and inline code. This element is most often used to render the content section of a page.
+
+```html
+<div class="markdown">
+	<p>Inner elements will be styled.</p>
+</div>
+```
+
+## Headings
+
+```markdown
+# Heading (level 1)
+
+## Headings (level 2)
+
+### Headings (level 3)
+
+#### Headings (level 4)
+
+##### Headings (level 5)
+
+##### Headings (level 6)
+```
+
+## Unordered list
+
+```markdown
+- First list item
+- Second list item
+  - First level of nested list
+    - Second level of nested list
+    - third level of nested list
+  - Second level again
+- Last one
+```
+
+## Ordered list
+
+```markdown
+1. First list item
+2. Second list item
+   1. First level of nested list
+      1. Second level of nested list
+      1. third level of nested list
+   1. Second level again
```

Comment:
Same as above. Right now it looks to me that it's actually at the same level as `First level of nested list`

Reply:
👍Looks like the text should be updated.